src/tests/ftest/osa/offline_extend.yaml

```diff
@@ -93,3 +93,6 @@ snapshot:
   test_with_snapshot: true
 test_ranks:
   rank_list: ["4", "5"]
+perform_osa_tasks:
+  exclude: "exclude"
+  drain: "drain"
```

Comment:
What's the point of these if the value is the same as the key?

Reply:
yes, will remove this option via YAML.